### PR TITLE
chore: update renovate config and CODEOWNERS

### PR DESCRIPTION
**Pull request by sandeeppanchal-bitgo**
Ticket: DO-3026

### CODE REVIEW

**Review by davidkaplanbitgo — APPROVED** [review]

File: CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# github actions
+.github                                             @BitGo/btc-team @BitGo/infra @BitGo/velocity
+renovate.json                                       @BitGo/btc-team @BitGo/infra @BitGo/velocity
+
+## THIS SHOULD ALWAYS BE LAST ##
+* @BitGo/btc-team
```
Comment (kchenye-bitgo) [inline CODEOWNERS:6]: same feedback as in https://github.com/BitGo/ecpair/pull/12

**Review by kchenye-bitgo — DISMISSED** [review]

File: CODEOWNERS [hunk 2]
```diff
@@ -0,0 +1,5 @@
+## THIS SHOULD ALWAYS BE FIRST ##
+* @BitGo/btc-team
+
+.github @BitGo/btc-team @BitGo/infra @BitGo/velocity
+renovate.json @BitGo/btc-team @BitGo/infra @BitGo/velocity
```
Comment (ericcrosson-bitgo) [inline CODEOWNERS:5]: Velocity does not need to own these files:

```suggestion
.github @BitGo/btc-team @BitGo/infra
renovate.json @BitGo/btc-team @BitGo/infra
```

Comment (sandeeppanchal-bitgo) [reply CODEOWNERS]: adjusted.

File: CODEOWNERS (hunk 1 above)
Comment (sandeeppanchal-bitgo) [reply CODEOWNERS]: adjusted.

**Pull request merged**